### PR TITLE
fix grammar mistakes locale si and add missing translations

### PR DESCRIPTION
**Pull request by madushanj**
I noticed few grammar mistakes on locale si (Sinhala Language) and missing translation. I fixed those grammar mistakes and added missing translations.

### CODE REVIEW

File: locales/si/validation-inline.php [hunk 1]
```diff
@@ -40,23 +40,23 @@
...
     'digits_between'       => 'මේ අතර විය යුතුයි :min හා :max ඉලක්කම්.',
     'dimensions'           => 'මෙම රූපය සාවද්ය මාන ඇත.',
-    'distinct'             => 'මේ ක්ෂේත්රයේ ඇති duplicate වටිනාකම.',
+    'distinct'             => 'මේ ක්ෂේත්‍රය ඇති duplicate වටිනාකම.',
     'email'                => 'මෙම විය යුතුය, වලංගු විද්යුත්-තැපැල් ලිපිනය.',
     'ends_with'            => 'මේ අවසන් විය යුතුය සමග එක් පහත සඳහන්: :values.',
     'exists'               => 'තෝරාගත් අගය සාවද්ය වේ.',
     'file'                 => 'අන්තර්ගත විය යුතුය ගොනුව.',
-    'filled'               => 'මේ ක්ෂේත්රයේ ඇති කළ යුතු අගය.',
-    'gt'                   => [
```
Comment (caouecs) [inline locales/si/validation-inline.php:49]: don't delete space please

File: locales/si/validation-inline.php [hunk 2]
```diff
@@ -40,23 +40,23 @@
...
-        'array'   => 'The content must have more than :value items.',
```
Comment (caouecs) [inline locales/si/validation-inline.php:50]: don't delete space, and don't update the sort, please

**Review by caouecs — CHANGES_REQUESTED** [review]

File: locales/si/validation-inline.php (hunk 1 above)
Comment (madushanj) [reply locales/si/validation-inline.php:49]: Sorry about that, probably due to automatic formatting. I'll fix it.

**Comment by caouecs** [comment]
Thank you

**Pull request merged**